docs/community-toolkit/hosting-java.md

```diff
@@ -38,9 +38,30 @@ Invoke-WebRequest `
 
 ---
 
+## Platform-specific considerations
+
+### Linux and macOS certificate trust
```

Comment:
This should be adjusted to not have two headings one after the other. Considering that there is only a single sub heading, I would rewrite the H2 to include the H3 info.